### DISCUSSION TITLE
Does anyone uses BME-SMD chips?

### DISCUSSION THREAD

**Discussion by Schluggi** [Schluggi/AIOsense#51] [Polls]
Related to https://github.com/Schluggi/AIOsense/issues/44, I consider to remove the support of the BME-SMD modules completely. Based on these assumptions:

- no real user-base
- problems with I²C master resistors (https://github.com/Schluggi/AIOsense/issues/44)
- hard to solder (heat gun needed)
- not upgradable
- afterwards we have more space for new features

Just to clarify: BME modules are still supported

Poll ends at 26th of march.

**Comment by florianmulatz** [comment]
I suggest to remove those. As you mentioned - I guess most "hobby elictritians" don't have the needed hardware to solder the small surface mounted pieces.

**Reply by florianmulatz** [reply]
The space might be used for a real co2 sensor for example MH-Z19C. (I know it's not the most accurate but it works quite reliable for me). 
Looks like the sensirion SCD30 (https://sensirion.com/de/produkte/katalog/SCD30) is too big for this PCB the smaller successors are again to solder with a heatgun - so nothing for this one :)

**Reply by Schluggi** [reply]
I agree :)

Feel free to create a [feature request](https://github.com/Schluggi/AIOsense/issues/new?assignees=Schluggi&labels=feature+request&template=feature_request.md&title=) for this. But [ESPHome](https://esphome.io/components/sensor/mhz19.html) only supports readouts via UART. So we can't use mmWave and CO² at the same time.

**Comment by cptbucky** [comment]
I don't use the surface mount components, however if I knew how to get JLPCB to install them for me then I'd love to shrink the overall package down. Perhaps a future iteration will have the option of a number of components that are surface mount.

**Reply by Schluggi** [reply]
An all-SMD version is already planned ;)
